example_configs/presto.yml

```diff
@@ -0,0 +1,41 @@
+rules:
+  - pattern: "com.facebook.presto.hive<type=NamenodeStats, name=hive><>(.+AllTime.+): (.*)"
+    name: "presto hive_namenodestats_$1"
+    help: "Presto Hive: NameNodeStats $1"
+    type: GAUGE
+  - pattern: "com.facebook.presto.hive<type=NamenodeStats, name=hive><>(.+TotalCount.*): (.*)"
+    name: "presto hive_namenodestats_$1"
+    help: "Presto Hive: NameNodeStats $1"
+    type: COUNTER
+  - pattern: "com.facebook.presto.hive.metastore.thrift<type=ThriftHiveMetastore, name=hive><>(.+AllTime.+): (.*)"
```

Comment:
This is pulling in the percentile too. You can also remove the "alltime" from the output, as it doesn't add anything.